HW_7/JS_HW_7_Task1.js

```diff
@@ -0,0 +1,81 @@
+/*
+1. Бесконечные аргументы
+  - Напишите функцию, принимающую на вход любое количество массивов
+  - Функция возвращает массив, в котором будут находиться все переданные в функцию в виде чисел
+  - Например: mergeArrays([1,2], [3,4], [5,6]) // [1,2,3,4,5,6]
+  - Решить с использованием Spread operator
+*/
+
+function getMergedArrays (...arrays) { 
+  return [].concat(...arrays);  
+}
+
+console.log(getMergedArrays([1,2,3,4],[6,7,8,9,10],[12,13,14,15],[16],[17,18,18.5,19]));
+
+
+
+nextTask()
+
+/*
+2. Devide by _
+  - Написать функцию, которая преобразует любое предложение в вот_Такой_Вот_Вид и возвращает его. 
+  - Первое слово должно начинаться с буквы в нижнем регистре, у остальных -  верхнем. 
+  - Пример: I am super engineer => i_Am_Super_Engineer
+*/
+
+function getDevidedBy_(sentence) {
+    const words = sentence.split(' ');
+    const devideBy_sentence = words.map((word, i) => {
+      if (i === 0) {
+        return word.charAt(0).toLowerCase() + word.slice(1);
+    } else {
+        return word.charAt(0).toUpperCase() + word.slice(1);
+    }
+  }).join('_');
```

Comment:
Изначально приводим в нижний регистр для подобньіх случаев `bROtHeR`
```suggestion
    const words = sentence.toLowerCase().split(' ');
    return words.map((word, i) => {
      if (i !== 0) {
        return word.charAt(0).toUpperCase() + word.slice(1);
    }
        return word;
  }).join('_');
```

Reply:
Принял. Задание, не верно понял, решил, что речь идет исключительно про первые буквы слов.